modules/alloy/src/test/ts/browser/api/events/AlloyTriggersRetargetTest.ts

```diff
@@ -0,0 +1,101 @@
+import { Chain, Logger, Mouse, UiFinder } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { Classes, EventArgs, SelectorFind } from '@ephox/sugar';
+
+import * as AddEventsBehaviour from 'ephox/alloy/api/behaviour/AddEventsBehaviour';
+import * as Behaviour from 'ephox/alloy/api/behaviour/Behaviour';
+import * as GuiFactory from 'ephox/alloy/api/component/GuiFactory';
+import * as AlloyEvents from 'ephox/alloy/api/events/AlloyEvents';
+import * as AlloyTriggers from 'ephox/alloy/api/events/AlloyTriggers';
+import * as NativeEvents from 'ephox/alloy/api/events/NativeEvents';
+import * as GuiSetup from 'ephox/alloy/api/testhelpers/GuiSetup';
+
+UnitTest.asynctest('AlloyTriggersRetargetTest', (success, failure) => {
```

Comment:
Ideally all new tests should use our preferred style, yes.

Reply:
This is mostly a result of getting more familiar with BDD testing as I progressed through writing tests. Because this is a completely new test, I'd agree that it should be BDD.

- [ ] make BDD style.